packages/x-charts/package.json

```diff
@@ -70,11 +70,13 @@
   "exports": {
     ".": {
       "types": "./index.d.ts",
-      "import": "./esm/index.js"
+      "import": "./esm/index.js",
+      "default": "./esm/index.js"
```

Comment:
Was this rule necessary? 🤔 
I might be mistaken, but codesandbox seemed to work after removing the `require` part. 🤔

Reply:
Yes, the setup of https://github.com/mui/mui-x/issues/10257 doesn't resolve to anything without a default condition. We could also decide to only have a `default` condition. But I kind of like how this also signals that `default` contains ESM.

Reply:
Thanks for clarifying. 👌 
I'll update my PR to only include the documentation update. 😉